src/metabase/driver/bigquery.clj

```diff
@@ -472,8 +472,9 @@
           ;; people can manually specifiy "foreign key" relationships in admin and everything should work correctly.
           ;; Since we can't infer any "FK" relationships during sync our normal FK tests are not appropriate for BigQuery, so they're disabled for the time being.
           ;; TODO - either write BigQuery-speciifc tests for FK functionality or add additional code to manually set up these FK relationships for FK tables
-          :features              (constantly (when-not config/is-test?
-                                               #{:foreign-keys}))
+          :features              (constantly (merge #{:native-parameters}
```

Comment:
This should be added to the list of valid features in the docstring for `metabase.driver/features`.